### ISSUE TITLE
a better way to distribute filebeat by rpm

### ISSUE THREAD

**Issue by mmgithub123** [elastic/beats#29677]
i had create a topic on [https://discuss.elastic.co/c/beats](https://discuss.elastic.co/)，the url is [https://discuss.elastic.co/t/some-puzzle-about-filebeat-version7-x-rpm-installation/293312](https://discuss.elastic.co/t/some-puzzle-about-filebeat-version7-x-rpm-installation/293312) ，it seems like necessary to open a issue in here.

the general idea is rename the wrapper utils /bin/filebeat from filebeat 7.x rpm package.

we can see a lot of people have trouble ，when they are install filebeat use rpm or just quickstart in  [https://discuss.elastic.co/t/filebeat-changing-filebeats-path-for-its-configuration-module/236243](https://discuss.elastic.co/t/filebeat-changing-filebeats-path-for-its-configuration-module/236243) 
and 
[https://discuss.elastic.co/t/filebeat-configuration-module-path-issue/268871](https://discuss.elastic.co/t/filebeat-configuration-module-path-issue/268871)

but it is not their problem.

The problem is when we install filebeat use rpm follow this page ：[Filebeat quick start: installation and configuration | Filebeat Reference [7.16] | Elastic](https://www.elastic.co/guide/en/beats/filebeat/7.16/filebeat-installation-configuration.html) 
use the command：
curl -L -O https://artifacts.elastic.co/downloads/beats/filebeat/filebeat-7.16.2-x86_64.rpm
sudo rpm -vi filebeat-7.16.2-x86_64.rpm

the rpm will install a wrapper utils，in /bin directory。but there is nothing about this in our document  in [Directory layout | Filebeat Reference [7.16] | Elastic](https://www.elastic.co/guide/en/beats/filebeat/7.16/directory-layout.html)

so，a lot of people do not know this wrapper exist。

additional，in the quick start document ：
Step 4: Set up assetsedit
command：
filebeat setup -e

the filebeat program here is the /bin/filebeat , but a lot
of people will think it is /usr/share/filebeat/bin/filebeat .
because this two program have the same name，and there is no describe about /bin/filebeat in the directory-layou document  [Directory layout | Filebeat Reference [7.16] | Elastic](https://www.elastic.co/guide/en/beats/filebeat/7.16/directory-layout.html)

In summary，i suggest ：
1，in the directory-layou document page add the describe about utils program /bin/filebeat
or
2,change the utils program name，we can call it filebeat-utils,so
people will not puzzle with /usr/share/filebeat/bin/filebeat
3,add some describe in quick start，so the reader will know the filebeat in command line is used /bin/filebeat not /usr/share/filebeat/bin/filebeat

thinks. 

@Filebeat
